### DISCUSSION TITLE
Controller + control don't trigger validation.

### DISCUSSION THREAD

**Discussion by wAISw** [react-hook-form/react-hook-form#4991] [Q&A] [answered]
Hi! 
Validation doesn't work at all for this code:
```
<Controller
  control={control}
  name="amount"
  defaultValue={0}
  rules={{ min: 10 }}
  render={({ field }) => <input id="amount" {...field} />}
/>
{errors.amount && <div>Maximum number of servings is 10.</div>}
```

The question is, how to validate fields like this?

**Comment by bluebill1049** [comment]
provide a codesandobx

**Comment by wAISw** [comment]
https://codesandbox.io/s/react-hook-form-get-started-forked-mkd24?file=/src/index.js

Before submitting it doesn't trigger validation, but if you submitted the form errors will work.

**Comment by wAISw** [comment]
It works! 
Thanks a lot!
Also I suggest you to add more examples to the documentation for dumb users like me :)

**Reply by bluebill1049** [reply]
you are not dumb mate, just miss what's in the doc. 👍 all good.

**Comment by bluebill1049** [comment, marked as answer]
have a read on this page: https://react-hook-form.com/api/useform

by default validation mode is `onSubmit`

<img width="1006" alt="Screen Shot 2021-04-28 at 7 42 37 pm" src="https://user-images.githubusercontent.com/10513364/116383100-e441b800-a859-11eb-9377-0868cd681a3b.png">
